### ISSUE TITLE
Crash after launching .exe from export

### ISSUE THREAD

**Issue by Yraian** [Zylann/godot_voxel#720]
Hello, 
First of all, thanks a ton for making this module, very useful.

- Terrain type - VoxelLODTerrain
- Mesher - VoxelMesherTransvoxel
- Generator - VoxelGeneratorGraph
- Stream - N/A (yet)
- Scene structure - Main scene (Node) -> Environment (Node) -> VoxelTerrain (VoxelLODTerrain)
- Plugin version (or where you got it) - v4.3.stable.custom_build [77dcf97d8]
- Godot version - ^
- OS - Windows

I have a project that I want to simply export to an EXE and send to some friends so they can play with it. The export occurs just fine, but when launching the .exe, I get a bunch of errors and the windows closes immediately.

<img width="491" alt="crash" src="https://github.com/user-attachments/assets/62ed894d-d560-432e-a720-415c8ccce430" />

A lot of these errors I know are probably caused by the first ones, but I'm not sure why it's happening at all. I'm not very skilled when it comes to reading these, but it says it's missing a Loader. Do I need a stream set up in order to play exported projects? Or am I missing something else here?

Sorry If I'm missing info, First time filling out something like this. Thanks for any help.

**Comment by Zylann** [comment]
You seem to have used the wrong export template. When you export a game with a version of Godot that contains a custom module, you also need an export template, which is basically that custom version of Godot without the editor. See https://voxel-tools.readthedocs.io/en/latest/getting_the_module/#exporting

**Comment by Yraian** [comment]
Aha! thank you so much for pointing me to where I was looking for in the docs, I knew it had to be in there, I just couldn't find it. 

It may be important to note in the docs, I could only get this to work *without* having any of the default templates installed, only the templates provided on your tool's release page. Additionally, I also ran into an issue where I had to uncheck "Export with debug" in the Save File menu right before exporting.

Thanks again for your help.
